tests/test_manifest_chunking.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
This is good, but one other thing we can consider is to modify one of the existing configs to have low value of in_memory_chunksize set directly there so that it's being tested in the e2e way as well. If you don't want to modify the configs (which is not the best thing to do, I agree), we can also change the e2e tests code to allow for additional config modifications same as we do for data prep. Then we don't have to change yaml configs, but can still patch a couple of processors when we load them for testing and allow arbitrary arguments changes

Reply:
Hmm, yes I also thought about allowing config modifications in e2e tests code, but I feel like it would add too much complexity to the code, and I can't think of any other scenarios when you would want to change the config just for the test. So I think a simple, separate test file is fine for now?

Reply:
Yeah, it's fine to only have your newly added test. That being said, there are actually other scenarios where we might want to change the configs - in many cases we have different options that users can select for the configs and we should ideally be testing multiple of them. So we can run the same config multiple times (with potentially different outputs), by providing different values of certain config parameters, such as e.g. PC vs no-PC for MLS or different speakers for English accents config.